e2e/logic/POM/navBarComponent.ts

```
@@ -69,14 +69,47 @@
 
     async clickOnSettingsBtn(): Promise<void> {
         await this.settingsButton.click();
-        await waitForTimeOut(this.page, 2000);
+        await waitForURL(this.page, urls.settingsUrl);
     }
 
-    async Logout(): Promise<void> {
+    async isSettingsButtonEnabled(): Promise<Boolean> {
```

Biome

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)